test/optimizer/applyDCEGraphRemovals.js

```diff
@@ -30,4 +36,4 @@ expI1;
 Module['expI2'];
 asm['expI3'];
 
-// EXTRA_INFO: { "unused": ["emcc$import$number", "emcc$import$name", "emcc$import$func", "emcc$export$expD4", "emcc$export$expI4"] }
+// EXTRA_INFO: { "unused": ["emcc$import$number", "emcc$import$name", "emcc$import$func", "emcc$export$expD4", "emcc$export$expD5", "emcc$export$expI4", "emcc$export$expI5"] }
```

Comment:
Please add D6, I6 which are used, that is, not in this list, and so they should not be removed (to test the code path where we check if used or not).